snapcraft/remote/git.py

```diff
@@ -110,7 +110,9 @@ def commit(self, message: str = "auto commit") -> None:
         target = [] if self._repo.head_is_unborn else [self._repo.head.target]
 
         try:
-            self._repo.create_commit("HEAD", author, author, message, tree, target)
+            return self._repo.create_commit(
```

Comment:
Would be nice to return the ID as a string rather than `pygit2.Oid` because this breaks through this wrapper's abstraction barrier by returning a `pygit2` object.

```suggestion
            return str(self._repo.create_commit(
```